release/charts/charts.go

```diff
@@ -93,3 +190,47 @@ func runChartsBuild(chartsRepoPath string, args ...string) ([]byte, error) {
 
 	return output, nil
 }
+
+// debugPullRequest will prompt the user to check the files and commits that will be pushed to the remote repository
+func debugPullRequest(r *git.Repository, remote, b string) error {
+	var (
+		execute bool
+	)
+
+	if execute = ecmExec.UserInput("Check files that will be pushed?"); execute {
```

Comment:
I'm not sure there's a need to assign the output from the UserInput function to a variable in any of the calls that assign to "execute".